src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -0,0 +1,227 @@
+      _output << boost::format("    %-22s: %d\n") % "Selector Lock Id" % rtp_node.second.get<uint16_t>("selector_lock_id");
+      _output << boost::format("    %-22s: %d\n") % "Selector Addr" %  rtp_node.second.get<uint64_t>("selector_address");
+      _output << boost::format("    %-22s: %d\n") % "Ping Buffer Row" % rtp_node.second.get<uint16_t>("ping_buffer_row");
+      _output << boost::format("    %-22s: %d\n") % "Ping Buffer Col" % rtp_node.second.get<uint16_t>("ping_buffer_column");
+      _output << boost::format("    %-22s: %d\n") % "Ping Buffer Lock Id" % rtp_node.second.get<uint16_t>("ping_buffer_lock_id");
+      _output << boost::format("    %-22s: %d\n") % "Ping Buffer Addr" % rtp_node.second.get<uint64_t>("ping_buffer_address");
+      _output << boost::format("    %-22s: %d\n") % "Pong Buffer Row" % rtp_node.second.get<uint16_t>("pong_buffer_row");
+      _output << boost::format("    %-22s: %d\n") % "Pong Buffer Col" % rtp_node.second.get<uint16_t>("pong_buffer_column");
+      _output << boost::format("    %-22s: %d\n") % "Pong Buffer Lock Id" % rtp_node.second.get<uint16_t>("pong_buffer_lock_id");
+      _output << boost::format("    %-22s: %d\n") % "Pong Buffer Addr" % rtp_node.second.get<uint64_t>("pong_buffer_address");
+      _output << boost::format("    %-22s: %b\n") % "Is Plrtp" % rtp_node.second.get<bool>("is_PL_RTP");
+      _output << boost::format("    %-22s: %b\n") % "Is Input" % rtp_node.second.get<bool>("is_input");
+      _output << boost::format("    %-22s: %b\n") % "Is Async" % rtp_node.second.get<bool>("is_asynchronous");
+      _output << boost::format("    %-22s: %b\n") % "Is Connected" % rtp_node.second.get<bool>("is_connected");
+      _output << boost::format("    %-22s: %b\n") % "Require Lock" % rtp_node.second.get<bool>("requires_lock");
+      count++;
+      _output << std::endl;
+    }
+    _output << std::endl;
+
+    count = 0;
+    for (auto& gmio_node : _pt.get_child("aie_metadata.gmios")) {
+      _output << boost::format("  %-4s: [%2d]\n") % "GMIO" % count;
+      _output << boost::format("    %-22s: %s\n") % "Id" % gmio_node.second.get<std::string>("id");
+      _output << boost::format("    %-22s: %s\n") % "Name" %gmio_node.second.get<std::string>("name");
+      _output << boost::format("    %-22s: %s\n") % "Logical Name" % gmio_node.second.get<std::string>("logical_name");
+      _output << boost::format("    %-22s: %d\n") % "Type" % gmio_node.second.get<uint16_t>("type");
+      _output << boost::format("    %-22s: %d\n") % "Shim_col" % gmio_node.second.get<uint16_t>("shim_column");
+      _output << boost::format("    %-22s: %d\n") % "Channel Number" % gmio_node.second.get<uint16_t>("channel_number");
+      _output << boost::format("    %-22s: %d\n") % "Stream Id" % gmio_node.second.get<uint16_t>("stream_id");
+      _output << boost::format("    %-22s: %d\n") % "Burst Length" % gmio_node.second.get<uint16_t>("burst_length_in_16byte");

```

Comment:
Help the user here in understanding the units regarding the "Burst Length".   Could the units be added?

Reply:
added